build/all-in-one/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN GOOS=$TARGETOS GOARCH=$TARGETARCH make build-timer
 RUN GOOS=$TARGETOS GOARCH=$TARGETARCH make build-gateway
 
 
-FROM ubuntu:22.04
+FROM --platform=$BUILDPLATFORM ubuntu:22.10
```

Comment:
what do you mean by adding `--platform=$BUILDPLATFORM`, vanus only builds one platform and doesn't support multi-platform.

Reply:
I failed to run the vanus image on the mac, after I added this tag, it works for me